kitchen.yml

```diff
@@ -0,0 +1,25 @@
+---
+driver:
+  name: vagrant
+
+verifier:
+  name: inspec
+
+provisioner:
+  name: chef_zero
+  deprecations_as_errors: true
+  chef_license: accept-no-persist
+
+platforms:
+  - name: amazonlinux-2
+  - name: centos-7
+  - name: centos-8
+  - name: debian-10
+  - name: debian-9
+  - name: ubuntu-18.04
+  - name: ubuntu-20.04
+
+suites:
+  - name: default
+    run_list:
+      - recipe[fastly::default]
```

Comment:
I think this should stay, but use the `recipe[fastly_test::service]` recipe instead. That way if someone has API keys and wants to actually test this, they could in some fashion.